manifests/a/AdoptOpenJDK/OpenJDK/17/17.0.0.18/AdoptOpenJDK.OpenJDK.17.locale.en-US.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: AdoptOpenJDK.OpenJDK.17
+PackageVersion: 17.0.0.18
+PackageLocale: en-US
+Publisher: AdoptOpenJDK
+PublisherUrl: https://github.com/AdoptOpenJDK/openjdk17-binaries
+PublisherSupportUrl: https://github.com/AdoptOpenJDK/openjdk17-binaries/issues
+Author: AdoptOpenJDK
+PackageName: AdoptOpenJDK JDK with OpenJ9 17

```

```suggestion
PackageName: AdoptOpenJDK JDK with Eclipse OpenJ9 17
```
This must match Control Panel.